rasa/core/training/story_writer/yaml_story_writer.py

```diff
@@ -124,7 +124,9 @@ def story_contains_forms(story_step) -> bool:
             `True` if the `story_step` contains at least one form action,
```

Comment:
Ok, this I'll fix as a separate commit, once this PR is merged 😄

Reply:
❤️